forum/users/tests.py

```diff
@@ -49,3 +51,39 @@ def test_password_recovery(self):
             status.HTTP_200_OK,
             "Status code is different from 200"
         )
+
+class LogoutAPITestCase(APITestCase):
+    def setUp(self):
+        self.user = CustomUser.objects.create_user(
```

Comment:
We already have something similar https://github.com/Project-Stage-Academy/UA1198ForumSB/blob/main/forum/forum/tests_setup.py#L7
Can you consider reusing it?